LBHackney-IT/lbh-fss-portal-frontend
# Feature/search user

**Description**

- Added user search functionality 
- Added pagination to table of users
- Added styling to table of users 

**Test instructions**

- pull this branch 
- navigate to users tab and compare users table to figma designs (check responsiveness)
- check user search functionality works as expected 
- check pagination at bottom of users table works as expected 

**Figma designs** 

https://www.figma.com/file/t6fanDKvLkCHJsY85ivFTF/Services-Directory?node-id=598%3A44959

![image](https://user-images.githubusercontent.com/69238887/92507353-c0fcc680-f1fe-11ea-8da1-a3dcd23ecc69.png)

**Ticket**

https://app.clickup.com/t/9e5u92
